frontend/src/App.js

```diff
@@ -14,6 +14,7 @@ import NotFoundPage from './pages/404Page';
 
 import Root,{loader as loadingAction} from './pages/Root';
 import Search from './pages/Search';
+import { DarkModeProvider } from './components/ToggleMode/DarkModeContext';
```

Comment:
Instead of having a specific DarkModeProvider, can we simply have a Theme provider that provides all the required colors? Based on the dark/light state, the colors will be displayed.